robottelo.properties.sample

```diff
@@ -49,6 +49,8 @@ ssh_key=
 # Webdriver logging options
 # A list of commands to be logged
 # log_driver_commands=newSession,windowMaximize,get,findElement,sendKeysToElement,clickElement,mouseMoveTo
+# The URL of container hosting repos on SatLab
```

Comment:
I think this comment needs more context to describe the configuration value.

How much of a URL does the user need to define here? protocol, host, and port?

Please reference `SatelliteQE/fedorapeople-repos` repo here as well, so there's an idea of where to get the content and how to host it.